sendReports.sh

```diff
@@ -20,8 +21,29 @@ processFile() {
    "http://localhost:8080/api/reports" -d "$(echo $POST_CONTENT)"
 }
 
+processFileNoPeriods() {
+   REPORT_CONTENT=$(envsubst "`printf '${%s} ' $(bash -c "compgen -A variable")`"  < $1)
+   REPORT_CONTENT="${REPORT_CONTENT//\"/\\\"}"
+   REPORT_NAME=$(echo $2 | sed 's/\([A-Z][^A-Z]\)/ \1/g')
+
+   POST_CONTENT=$(printf "$POST_TEMPLATE_NO_PERIODS" "$REPORT_NAME" "$REPORT_CONTENT")
```

Comment:
If is the only difference between processFileNoPeriods and processFile is the "POST_TEMPLATE" variable we should consider using this as the parameter.

Reply:
I think that it's a good idea. Thanks!